www/src/components/CodeBlock.jsx

```diff
@@ -49,6 +54,7 @@ function CodeBlock({ children, className, live }) {
             ...ParagonIcons,
             ...ParagonReact,
             useCallback,
+            useEffect,
```

Comment:
Does this just add `useEffect` to things we can use in the doc site codeblocks?

Reply:
Exactly 👍  I had an example using useEffect locally to test these changes with Voiceover (show spinner on click of a button), and just left this in there because figured it could be useful for someone using the live code editors on the documentation site.